spec/models/allocation_spec.rb

```diff
@@ -176,5 +176,13 @@
         expect(subject.days_until_finish).to eq('Finalizado')
       end
     end
+
+    context 'when allocation did not started' do
+      subject { build_stubbed(:allocation, end_at: Time.zone.today + 1.month) }
```

Comment:
just to let you know; you can also do `1.month.from_now`

Reply:
Nice.